File: app/src/main/java/com/yy/realx/RealXApplication.java
package com.yy.realx;

import android.support.multidex.MultiDexApplication;
import cn.jiguang.share.android.api.JShareInterface;

public class RealXApplication extends MultiDexApplication {
    @Override
    public void onCreate() {
        super.onCreate();
        JShareInterface.setDebugMode(true);
        JShareInterface.init(this);
    }
}

File: app/src/main/java/com/yy/realx/RealXApplication.kt
package com.yy.realx

import android.support.multidex.MultiDexApplication
import cn.jiguang.share.android.api.JShareInterface

class RealXApplication : MultiDexApplication() {
    override fun onCreate() {
        super.onCreate()
        JShareInterface.setDebugMode(true)
        JShareInterface.init(this)
    }
}
